README.md

```diff
@@ -1,4 +1,4 @@
-# ![jumble](https://github.com/theIYD/jumble-password/blob/master/media/logo.png)
+# ![jumble](https://github.com/Arslanshn/jumble-password/blob/master/media/Logo.png?raw=true)
```

Comment:
The picture link should be on the jumble password repository of the original author.